dataflow/gemma/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This uses Ubuntu with Python 3.11
+ARG SERVING_BUILD_IMAGE=tensorflow/tensorflow:2.15.0-gpu
+
+FROM ${SERVING_BUILD_IMAGE}
+
+WORKDIR /workspace
+
+RUN apt-get update -y && apt-get install -y \
+    cmake
+
+COPY requirements.txt requirements.txt
+RUN pip install --upgrade pip \
+    && pip install --no-cache-dir -r requirements.txt 
+
+# Copy files from official SDK image, including script/dependencies.
+COPY --from=apache/beam_python3.11_sdk:2.54.0 /opt/apache/beam /opt/apache/beam
+
+RUN pip install --no-cache-dir keras_nlp==0.8.0 
```

Comment:
is it possible to collapse the two pip calls to one layer?

Reply:
Potentially? I think keeping them separate avoids any potential for the dependency conflict to cause issues.

Reply:
updating to more recent dependenices let my put everything into requirements.txt